app/Files/File.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
Overengineering is a bad thing. That claim is arguable, it's not absolute. And that's not a perfect analogy. It can be argued that in this case the file upload section might not get more complicated. Doing more than sufficient for now is over-engineering. Your analogy assumes that it will always get more complicated.